sdk/identity/identity/test/internal/node/defaultAzureCredential.spec.ts

```diff
@@ -122,6 +123,15 @@ describe("create functions", () => {
     expect(cliSpy).not.toHaveBeenCalled();
     expect(devCliSpy).not.toHaveBeenCalled();
     expect(psSpy).not.toHaveBeenCalled();
+
+    expect(miSpy).toHaveBeenCalledWith({
+      allowInsecureConnection: true,
+      retryOptions: {
+        maxRetries: 5,
+        retryDelayInMs: 800,
+      },
+      sendProbeRequest: false,
```

Comment:
Just checking, do we have a test that tests the default scenario? When the user doesn't set `sendProbeRequest` and that value is true by default in DAC? @minhanh-phan

Reply:
I think that'd fall under the `msalMsiProvider.spec.ts` test when the value is set to true. I don't think we need to add an additional DAC test for true because at that point, we're validating the correctness of the JS syntax for this line `options.sendProbeRequest ??= true`